mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -1116,70 +1132,62 @@ def acknowledge(self):
         if not process:
             LOG.warning("Unable to play 'acknowledge' audio file!")
 
-    def init_dialog(self, root_directory):
-        # If "<skill>/dialog/<lang>" exists, load from there.  Otherwise
-        # load dialog from "<skill>/locale/<lang>"
-        dialog_dir = join(root_directory, 'dialog', self.lang)
-        if exists(dialog_dir):
-            self.dialog_renderer = load_dialogs(dialog_dir)
-        elif exists(join(root_directory, 'locale', self.lang)):
-            locale_path = join(root_directory, 'locale', self.lang)
-            self.dialog_renderer = load_dialogs(locale_path)
-        else:
-            LOG.debug('No dialog loaded')
```

Comment:
`init_dialog()` is part of the public skill api and needs to remain. At least until 20.08.